templates/exporter/deployment.yaml

```diff
@@ -0,0 +1,100 @@
+{{- if .Values.exporter.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
Cant we have this as a side-car for each of the components?

This way dosent look very kubernetes

Reply:
maybe, but why? it's worthless any efforts